LEGO1/mxioinfo.cpp

```diff
@@ -124,9 +124,9 @@ MxLong MXIOINFO::Read(void *p_buf, MxLong p_len)
 }
 
 // OFFSET: LEGO1 0x100cca00
-MxLong MXIOINFO::Seek(MxLong p_offset, int p_origin)
+MxResult MXIOINFO::Seek(MxLong p_offset, int p_origin)
```

Comment:
This should remain `MxLong`. If you look at the function you will see it assigns various values to `result` which don't necessarily have to be `SUCCESS` or `FAILURE`, so using `MxResult` as the return type would be misleading

Reply:
Sorry! I didn't notice this. I think I was tripped up by the `result` name.

Reply:
I'd just leave it `-1` and only use the constants in the context of the `MxResult` type (think C++11 `enum class` behavior)